src/ceph-volume/ceph_volume/util/disk.py

```diff
@@ -603,6 +603,12 @@ def __div__(self, other):
         _b = self._b / other
         return Size(b=_b)
 
+    def __bool__(self):
+        return self.b != 0
+
+    def __nonzero__(self):
```

Comment:
Isn't `__nonzero__()` py2 equivalent of `__bool__()`? Are we still supporting py2?

Reply:
> Are we still supporting py2?
 We are in `nautilus`.